### ISSUE TITLE
Adjust the catalog/report change confirm dialog text to warn user of data destruction when change specific order of catalogs

### ISSUE THREAD

**Issue by dmundra** [GSA/openacr#344]
From https://github.com/GSA/openacr-editor/pull/55

> Data loss really should only occur if you are going from
> 
> * 508 or 508 + WCAG  --> WCAG
> * 508 + WCAG  --> 508
> 
> Should we warn folks if they are going from:
> 
> * WCAG  --> 508 + WCAG
> * 508 --> 508 + WCAG
> 
> I think I'm missing one...
> 
> Maybe we could congratulate them, but warn them that they might loose data if they go back? Not sure. That's probably an enhancement.
